src/Data/DatabaseUuidIdGenerator.php

```diff
@@ -50,7 +52,7 @@ public function __construct(\Database $database)
      */
     public function generate()
     {
-        return $this->database->query('SELECT UUID() as id')->first()->id;
+        return $this->connection->query('SELECT UUID() as id')->fetch(\PDO::FETCH_OBJ)->id;
```

Comment:
Could be  `return $this->connection->query('SELECT UUID() as id')->fetch(\PDO::FETCH_COLUMN)`?

Reply:
Has been change.

I have change this in https://github.com/contao-community-alliance/dc-general/pull/386/commits/12ba45fe49de01e41b02cb6ada1f1ef1a6e7a9a0#diff-409a5405b81668016fbcbc5c8a4ae196R461 and https://github.com/contao-community-alliance/dc-general/pull/386/commits/12ba45fe49de01e41b02cb6ada1f1ef1a6e7a9a0#diff-409a5405b81668016fbcbc5c8a4ae196R717 too.